src/main/scala-2.11/com/linagora/gatling/imap/protocol/command/MessageRange.scala

```diff
@@ -0,0 +1,30 @@
+package com.linagora.gatling.imap.protocol.command
```

Comment:
that's never too late when you are writing a PR. Could be fine if you have put the good licence, but that's not the case. See also https://github.com/linagora/gatling-imap/pull/14 which is still not merged.